src/views/AddRecord.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from "react";
-import { Text, View, ScrollView, TouchableOpacity } from "react-native";
+import { Text, ScrollView, TouchableOpacity, Button, Image, View, Platform, ActivityIndicator, Alert} from "react-native";
```

Comment:
Did you use a code formatter for this?

Reply:
yes but i forgot to apply it i am using prettier